src/org/plovr/Compilation.java

```diff
@@ -363,6 +363,10 @@ public void writeCompiledCodeToFiles(
         outputFile = new File(outputFile.getParentFile(), fileName);
       }
 
+      if ( config.getAddSourceMapUrl() ) {
+        moduleCode += "\n//# sourceMappingURL=" + sourceMapPath + "_" + moduleName + "\n";
```

Comment:
hmmm...this doesn't seem right? sourceMapPath is a file path, but here you're using it as a URL path? 

maybe, instead of a `boolean` property, the config should specify a URL root?